p/poetry/poetry_ubi_9.3.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package          : poetry
+# Version          : 1.8.3
+# Source repo      : https://github.com/python-poetry/poetry.git
+# Tested on        : UBI:9.3
+# Language         : Python
+# Travis-Check     : True
+# Script License   : Apache License, Version 2 or later
+# Maintainer       : Aastha Sharma <aastha.sharma4@ibm.com>
+#
+# Disclaimer       : This script has been tested in root mode on given
+# ==========         platform using the mentioned version of the package.
+#                    It may not work as expected with newer versions of the
+#                    package and/or distribution. In such case, please
+#                    contact "Maintainer" of this script.
+#
+# ---------------------------------------------------------------------------
+
+#variables
+PACKAGE_NAME=poetry
+PACKAGE_VERSION=${1:-1.8.3}
+PACKAGE_URL=https://github.com/python-poetry/poetry.git
+
+#install dependencies
+yum install -y --allowerasing python-pip python-devel git gcc gcc-c++ make curl openssl openssl-devel wget openssl-devel bzip2-devel libffi-devel zlib-devel
+
+# Create a symbolic link for python3
+ln -s /usr/bin/python3 /usr/bin/python
```

Comment:
Avoid creating symbolic link.

Reply:
> Avoid creating symbolic link.

Right, but here in this case it was required since some of the testcase needed the path /usr/bin/python